### ISSUE TITLE
NPP

### ISSUE THREAD

**Issue by ktoddbrown** [ktoddbrown/PrecipOnSOC#3]
I've been trying to figure out how to get the MODIS NPP data product and I'm having a devil of a time. Any suggestions @bpbond ?

**Comment by bpbond** [comment]
Not https://modis.gsfc.nasa.gov/data/dataprod/mod17.php ?

I've downloaded and used MODIS GPP before, but not NPP.

**Comment by ktoddbrown** [comment]
Right. I spent about an hour fighting with this download. I was hoping you had a magic link :( I'll poke around a bit more and ask around.

**Comment by ktoddbrown** [comment]
9a33f70661724706584d638a82ed111f28af9a15 Found the following NPP product with help from Bill Smith http://files.ntsg.umt.edu/data/NTSG_Products/MOD17/GeoTIFF/MOD17A3/GeoTIFF_30arcsec/MOD17A3_Science_NPP_mean_00_15.tif He's been hard to nail down so I don't have the proper citation for this yet but this is pretty much exactly what we are looking for. I have some questions about some of the default values that I need to talk with him about in more detail.
